demo/MINDCUB3R/README.md

```diff
@@ -1,26 +1,13 @@
 # MINDCUB3R
 
 ### Installation
-This can take a minute on an EV3
-* cd ~/ev3dev-lang-python/demo/MINDCUB3R/kociemba/ckociemba/
-* sudo apt-get update
-* sudo apt-get install build-essential
-* make
-* sudo make install
-
-The first time you run this it creates a cache directory, this takes about 30s on an EV3
-
-* cd ~/ev3dev-lang-python/demo/MINDCUB3R/
-* kociemba DRLUUBFBRBLURRLRUBLRDDFDLFUFUFFDBRDUBRUFLLFDDBFLUBLRBD
-
+This can take several minutes on an EV3
+```
+sudo apt-get install build-essential python3-pip libffi-dev
+sudo pip3 install git+https://github.com/dwalton76/kociemba.git
+sudo pip3 install git+https://github.com/dwalton76/rubiks-color-resolver.git
+kociemba DRLUUBFBRBLURRLRUBLRDDFDLFUFUFFDBRDUBRUFLLFDDBFLUBLRBD
```

Comment:
It does look like greek doesn't it :)  The string is a representation of the colors of each of the 54 squares of a 3x3x3 cube.  So the D at the beginning means that square #1 is the same color as the middle square of the Down side (the bottom), the R means that square #2 is the same color as the middle square of the Right side, etc.  The kociemba program takes that color data and returns a sequence of moves that can be used to solve the cube.  Just to keep things consistent the syntax for the sequence of moves also looks like greek
```
robot@beaglebone[lego-crane-cuber]# kociemba DRLUUBFBRBLURRLRUBLRDDFDLFUFUFFDBRDUBRUFLLFDDBFLUBLRBD
D2 R' D' F2 B D R2 D2 R' F2 D' F2 U' B2 L2 U2 D R2 U
robot@beaglebone[lego-crane-cuber]# 
```

I'll add some of this text to the README